pkg/handlers/v1/notification_test.go

```diff
@@ -50,7 +52,9 @@ func TestHandle(t *testing.T) {
 				{
 					ScanID:    "1",
 					SiteID:    "11",
-					Timestamp: ts.Add(time.Second),
+					ScanType:  "Scheduled",
+					StartTime: ts.Add(time.Second),
```

Comment:
Question: Don't you mean to substract a second, and not add one? That way in this successful case, the `StartTime` is not after `ts` (now)

Reply:
StartTime doesn't matter for these tests; scan-notifier's logic only involves the EndTime. The only reason I give it any specific value here is because it doesn't make sense to have a start time after the end time. This way the tests should require less refactoring if we ever *start* to consider StartTime.